1-JavaScript/Week4/homework/ex1-doubleEvenNumbers.test.js

```diff
@@ -11,13 +11,8 @@ Let's rewrite it (or _refactor_ it, as experienced developers would call it):
 ------------------------------------------------------------------------------*/
 // ! Function to be tested
 function doubleEvenNumbers(numbers) {
-  // TODO rewrite the function body using `map` and `filter`.
-  const newNumbers = [];
-  for (let i = 0; i < numbers.length; i++) {
-    if (numbers[i] % 2 === 0) {
-      newNumbers.push(numbers[i] * 2);
-    }
-  }
+  let newNumbers = [];
+  newNumbers=numbers.filter(number=>number%2===0).map(number=>number*2);
```

Comment:
```suggestion
  newNumbers= numbers.filter(number => number % 2 === 0).map(number => number * 2);
```
We're not optimising for memory anymore but for readability, so use white space please :)

Reply:
i closed prettier extension because of some setting about homework and i forgot to open again. I changed them.